airbyte-ci/connectors/metadata_service/orchestrator/tests/fixtures/cloud_registry.json

```diff
@@ -8743,7 +8743,7 @@
       "sourceDefinitionId": "e7778cfc-e97c-4458-9ecb-b4f2bba8946c",
       "name": "Facebook Marketing",
       "dockerRepository": "airbyte/source-facebook-marketing",
-      "dockerImageTag": "0.2.86",
+      "dockerImageTag": "0.4.3",
```

Comment:
These files are edited automatically please rollback the changes here you only need to change the version in the connector folder.